src/features/defi/components/DefiModalHeader/DefiModalHeader.tsx

```diff
@@ -0,0 +1,19 @@
+import { Box } from '@chakra-ui/layout'
+import { Image } from '@chakra-ui/react'
+import { Text } from 'components/Text'
+
+type DefiModalHeaderProps = {
+  headerImageSrc: string
+  headerText: string | [string, Record<string, string>]
+}
+
+export const DefiModalHeader = ({ headerImageSrc, headerText }: DefiModalHeaderProps) => (
+  <>
+    <Box textAlign='center'>
+      <Image src={headerImageSrc} width='68px' height='73px' />
```

Comment:
I think you should add an alt, usefull for screen readers

It works because the image is pretty small, but usually using fixed width and heights can lead to a lot of responsive problems

I think you can let the height natural, the width should be enough isn't it?